Cygwin is missing strverscmp(3), but this function isn't used anyway

--- origsrc/vlc-1.0.0/modules/visualization/galaktos/tree_types.c	2009-05-10 18:03:55.000000000 -0500
+++ src/vlc-1.0.0/modules/visualization/galaktos/tree_types.c	2009-07-17 01:13:37.547808800 -0500
@@ -48,12 +48,14 @@ int compare_string(char * str1, char * s
 	
 }	
 
+#if 0
 /* Compares a string in version order. That is, file1 < file2 < file10 */
 int compare_string_version(char * str1, char * str2) {
 
   return strverscmp(str1, str2);
 
 }
+#endif
 
 
 void free_int(void * num) {
--- origsrc/vlc-1.0.0/modules/visualization/galaktos/tree_types.h	2009-05-10 18:03:55.000000000 -0500
+++ src/vlc-1.0.0/modules/visualization/galaktos/tree_types.h	2009-07-17 01:13:18.330265100 -0500
@@ -7,4 +7,6 @@ void free_string(char * string);
 void * copy_int(int * num);
 void * copy_string(char * string);
 
+#if 0
 void * compare_string_version(char * str1, char * str2);
+#endif
